### PR TITLE
Modules 3298

### PR DESCRIPTION
**Pull request by ThoughtCrhyme**
Added beaker-hostgenerator to dynamically create the configs for acceptance and reference testing.

### CODE REVIEW

**Comment by ferventcoder** [comment]
Something is wrong with the PR. Looks like you need to update/rebase your branches?

File: Gemfile [hunk 1]
```diff
@@ -47,8 +47,9 @@ end
 
 group :system_tests do
   gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.20')
-  gem 'master_manipulator', '~> 1.2',  :require => false
-  gem 'beaker-windows', '~> 0.6',      :require => false
+  gem 'master_manipulator', '~> 1.2',   :require => false
+  gem 'beaker-windows', '~> 0.6',       :require => false
+  gem 'beaker-hostgenerator', '~> 0.6', :require => false
```
Comment (ferventcoder) [inline Gemfile]: Update .sync.yml as well for this.

File: Rakefile [hunk 2]
```diff
@@ -2,6 +2,7 @@ require 'rake'
 require 'rspec/core/rake_task'
 require 'puppetlabs_spec_helper/rake_tasks'
 require 'puppet'
+require 'beaker-hostgenerator'
```
Comment (cowofevil) [inline Rakefile]: This is not necessary in the Rake file as you're not actually importing any code from BHG.

Comment (cowofevil) [inline Rakefile]: Also, this is why TravisCI is failing.

Comment (ThoughtCrhyme) [inline Rakefile]: Ok, I'll take it out, thanks Ryan.

**Pull request closed without merging**